pkgs/os-specific/linux/thunderbolt/default.nix

```diff
@@ -23,11 +24,13 @@ stdenv.mkDerivation rec {
     txt2tags
   ];
 
-  cmakeFlags = [
-    "-DCMAKE_BUILD_TYPE='Release'"
-    "-DUDEV_BIN_DIR=$out/bin"
-    "-DUDEV_RULES_DIR=$out/udev"
-  ];
+  # These can't go in the normal nix cmakeFlags because $out needs to be
```

Comment:
See also https://github.com/NixOS/nixpkgs/pull/37693

Reply:
Is it okay to use placeholder expressions in nixpkgs now?

Reply:
Unfortunately not :cry: